src/metabase/notification/core.clj

```diff
@@ -41,15 +42,17 @@
   "Send the notification to all handlers synchronously."
   [notification-info :- NotificationInfo]
   (let [noti-handlers (hydrate-notification-handler (t2/select :model/NotificationHandler :notification_id (:id notification-info)))]
-    (log/infof "[Notification %d] Found %d handlers" (:id notification-info) (count noti-handlers))
+    (log/infof "[Notification %d] Found %d %s"
+               (:id notification-info) (count noti-handlers) (u/format-plural (count noti-handlers) "handler" "handlers"))
```

Comment:
It might be nice to overload `format-plural` to "just add s" if the plural string is omitted.

Then again, I could see copy paste editing leading to bugs that way.